api/client.yaml

```
@@ -1513,6 +1513,249 @@ paths:
+  /customers/{customerID}/representatives:
+    post:
+      tags: [Customers]
+      summary: Add customer representative
+      description: Add a Customer Representative
+      operationId: addCustomerRepresentative
+      parameters:
+        - name: X-Request-ID
+          in: header
+          description: Optional requestID allows application developer to trace requests through the systems logs
+          example: rs4f9915
+          required: true
+          schema:
+            type: string
+        - name: X-Organization
+          in: header
+          description: Value used to separate and identify models
+          example: org342
+          schema:
+            type: string
+        - name: customerID
+          in: path
+          description: customerID of the Customer for whom to add the representative
+          required: true
+          schema:
+            type: string
+            example: e210a9d6-d755-4455-9bd2-9577ea7e1081
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateCustomerRepresentative'
+      responses:
+        '200':
+          description: A customer object
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Customer'
+        '400':
+          description: Customer representative was not added, see error(s)
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives/{representativeID}:
+    put:
+      tags: [Customers]
+      summary: Update customer representative
+      description: Updates the specified customer representative
+      operationId: updateCustomerRepresentative
+      parameters:
+        - name: customerID
+          in: path
+          description: Customer ID
+          required: true
+          schema:
+            type: string
+            example: e210a9d6-d755-4455-9bd2-9577ea7e1081
+        - name: representativeID
+          in: path
+          description: Representative ID
+          required: true
+          schema:
+            type: string
+            example: 1d62e297-9727-4084-a902-1031da932c9e
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateCustomerRepresentative'
+      responses:
+        '200':
+          description: Updated Customer representative
+        '400':
+          description: See error message
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+    delete:
+      tags: [Customers]
+      summary: Delete a customer's representative
```

I changed summaries to read more like the following.

```suggestion
      summary: Delete Customer Representative
```